sdk/easm/azure-defender-easm/samples/sample1_managing_discovery_runs.py

```diff
@@ -0,0 +1,41 @@
+'''
+This sample demonstrates how to create and manage discovery runs in a workspace using the discovery_groups module of the EasmClient
```

Comment:
Also not blocking right now - but before GA we should make sure we have the equivalent samples and tests in async as well.

Reply:
I'll ensure they're there before GA